docs/Containers/Container-Tasks.md

```diff
@@ -0,0 +1,298 @@
+
+A new tab has been added to the project settings page for container
+configurations. Users can create a list of resource configuration
+presets that can be referenced by name in the size field of their
+container YAML configurations.
+
+![containers.png](../images/container_configuration.png)
+
+Options:
+
+-   Name: The name for the resource preset. Names must be unique within
+    the list.
+
+-   Memory: The amount of memory (in MiB) to allocate.
+
+-   CPU: The CPU units the container can use. These values are expressed
+    in 'vCPU Units'. 1024 CPU units is the equivalent of 1vCPU.
+
+Users can define as many container configurations as needed, reflecting
+different appropriate resource needs for various tasks.
+
+### **Task Metadata**
+
+A link to a container task's respective container replaces the typical
+host link.
+
+<img alt="containerized_task_metadata.png" height="400" src="../images/container_metadata.png" width="300"/>
+
+### **Container Page**
+
+The link in the task metadata sidebar takes you to the container page,
+which details the lifecycle of a container and their tasks. Like the
+host page, event logs exist charting the journey of a container task
+from initialization to termination. Task events such as container
+assignment and status changes are also recorded, as well as the clearing
+of a task from a container once it has run its course.
+
+![container_event_logs.png](../images/container_page.png)
+
+## Other Considerations
+
+### Disk Space
+
+For tasks that require a large amount of disk space, please be aware
+that container tasks may **not** be the best fit for your use case at this
+time.
+
+Each instance is provisioned with 200GB of space; however, the actual
+disk space available for each container task can vary depending on the
+number and the nature of tasks sharing the same instance. In this sense,
+while CPU and memory are isolated allocations to each container, disk
+space is a shared resource across all containers on the same host. As
+such, dedicating more memory and CPU for a container makes it less
+likely to share the instance's disk space with other containers.
+Conversely, less resource-hungry containers are more likely to share the
+instance with others, so they will likely have a smaller share of the
+available disk space.
+
+In practical terms, this means that while each instance has a maximum of
+200GB of disk space, please bear in mind that the effective disk space
+available to your container tasks might be less and fluctuate, as other
+containers take and release disk space as they get created and exit,
+respectively. While we work on a more robust solution to this notion of
+isolating disk space, we recommend that you keep your container task
+disk space usage to a maximum of 10GB.
+
+### Git Cloning Modules With SSH
+
+In a containerized environment, cloning modules via SSH is unsupported in order to prevent
+all teams from sharing the same SSH keys on the container instance hosts.
```

Comment:
"sharing the same SSH keys in their containers" since the hosts' SSH keys don't have any relevance to the running containres.